app/routes/admins.routes.js

```diff
@@ -1,22 +1,27 @@
 var admins = require('../controllers/admins.controller');
+var users = require('../controllers/users.controller');
 var auth = require('../utils/auth-middleware.utils');
 
 module.exports = function (app) {
     app.post('/admin/register',
         auth.authenticateTokenMiddleware,
-        admins.isAdmin,
-        admins.findUserByEmail,
+        admins.isAdminMiddleware,
+        users.findUserByEmail,
         admins.doesAdminAlreadyExist,
         admins.registerAdmin);
 
+    app.get('/isAdmin',
```

Comment:
Actually, one small change I'd make is to change this route to '/is-admin' or '/admin/is-admin' to match our current naming conventions.